newsletter-january-2023.rst

```diff
@@ -0,0 +1,118 @@
+.. post:: Jan 1, 2023
+   :tags: newsletter, python
+   :author: Ben
+   :location: MLM
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from the last month,
+      current focus, and upcoming features.
+
+Read the Docs newsletter - January 2023
+=======================================
+
+Happy 2023!
+
+News and updates
+----------------
+
+Here are the latest updates from our team since the :doc:`previous newsletter </newsletter-december-2022>`:
+
+- 📹️ Eric delivered a talk at DjangoCon US 2022 with practical tips for developing *state of the art* documentation projects.
+  You can watch it here: `Documenting Django Code in 2022`_
+- 🛠️ In line with the talk,
+  we continued reorganizing our own documentation to follow the `Diátaxis Framework`_.
+- 🚢️ Sphinx 6.0 and 6.1 have been released.
+  If you want things to not break,
+  it's good to wait a bit.
+  Read :doc:`our considerations for upgrading </sphinx6-upgrade>`.
+- 🔒️ We fixed a security vulnerability on our platform. See: `GHSA-368m-86q9-m99w`_
+
+You can always see the latest changes to our platforms in our :doc:`Read the Docs Changelog <readthedocs:changelog>`.
+
+.. _Documenting Django Code in 2022: https://www.youtube.com/watch?v=mqn0D4xat58
+.. _Diátaxis Framework: https://diataxis.fr/
+.. _GHSA-368m-86q9-m99w: https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-368m-86q9-m99w
+
+Upcoming features
+-----------------
+
+..
+  Note:
+  
+  When creating newsletter drafts, we keep the items here from the previous newsletter.
+  This is in order to ensure due follow-up on features that are announced publicly.
+  
+  Feature done? A great follow-up is to add what was previously an upcoming feature as a released feature in the former section.
+  
+  Feature not done?
+  Make sure that upcoming features are announced with a link to issues or PRs where the progress can be seen.
+  If this is done, then subsequent newsletters aren't compelled to share progress when it's uninteresting.
+  
+  If a feature was announced as upcoming but isn't yet released,
+  then try rephrasing the announcement as a general news update about the progress and where it can be followed.
+
+The core team has laid out its 2023 Q1 roadmap and our general focus will be on:
+
+- Improving support for using *any* documentation tool in the build process.
+- Currently, users can either customize a default build process or overwrite the entire process from scratch.
+  We are aligning these two approaches,
+  making it easier to design and configure builds for a project's individual needs.
+- Continuing to restructure and rewrite the user documentation of Read the Docs.
+  We look forward to publishing the restructured documentation which is likely happening at the end of Q1.
+- Dashboard redesign: The entire Dashboard will be redesigned and re-implemented.
+  This work is about to be launched!
```

Comment:
Yeah, I don't know if makes sense mentioning it _again_. It seems like "old news" to me. We've been promoting this work since April 2021: https://github.com/readthedocs/blog/blob/d627e9fc6e183ff40c8f152375ec42c563daaa81/newsletter-april-2021.rst?plain=1#L91-L94

If we want to mention this, we should probably rephrase it to say that we are going to resume working on this or similar.

Reply:
It's funny though how @ericholscher's formulation is so consistent with almost 2 years span - "we hope to launch in public beta soon" and "we are hoping to have a public beta in Q1"

It does seem like a good thing to know what the difference is "this time" from the previous time that it was mentioned. I imagine that we have become a lot better at planning? So perhaps there is a bit more "confidence" behind this announcement.

I'll remove the beta announcement. I'll just write that we are working towards a public beta and not anything about when it's launched?